pkgs/applications/virtualization/docker/default.nix

```diff
@@ -11,14 +11,14 @@ with lib;
 
 stdenv.mkDerivation rec {
   name = "docker-${version}";
-  version = "1.13.1";
-  rev = "092cba3"; # should match the version commit
+  version = "17.03.0-ce";
+  rev = "60ccb22"; # should match the version commit
```

Comment:
It's used [here](https://github.com/NixOS/nixpkgs/pull/23872/files#diff-7eb1506cdd2e993fea63f74a11fbc75bL83) but yeah the commit I put could have been better 😅